user.tf

```diff
@@ -1,5 +1,5 @@
 module "ci_user" {
-  source = "github.com/cisagov/ci-iam-user-tf-module"
+  source = "github.com/cisagov/ci-iam-user-tf-module?ref=improvement%2Fuse_tf_aws_3.x_provider"
```

Comment:
Don't forget to change this back!

Reply:
Reverted with https://github.com/cisagov/molecule-iam-user-tf-module/pull/17/commits/e55e6efc85be9ad7684e5afc65b2a2e7fab0e5ce